libs/R-TypeLogic/Server/Systems/EnemyShootSystem.cpp

```diff
@@ -5,13 +5,13 @@
 ** EnemyShootSystem
 */
 
-#include <chrono>
 #include "EnemyShootSystem.hpp"
-#include "GameComponents/EnemyComponent.hpp"
-#include "GameComponents/ShootingFrequencyComponent.hpp"
-#include "GameEntityManipulation/CreateEntitiesFunctions/CreateEnemyProjectile.hpp"
-#include "GameSharedResources/GameClock.hpp"
+#include <chrono>
```

Comment:
Not clang format, please put system library above everything

Reply:
I formatted the code at each save, I don't know how it remains